translations/catroweb.en.yml

```
@@ -64,13 +64,31 @@ deleteAllNotificationsButtonTitle: "Delete all"
 notificationsClearError: "Unable to mark notifications as read!"
 notificationsDeleteError: "Unable to delete notifications!"
 pleaseWait: "Please wait..."
+followError: "Unable to follow the desired user!"
+unfollowError: "Unable to unfollow the desired user!"
+unfollowQuestion: "Are you sure you want to unfollow this user?"
+unfollowButton: "Unfollow"
+cancelButton: "Cancel"
 
 indexPage:
   recommendedSection:
     info:
       title: 'What projects are recommended here?'
       description: 'The more reactions (Thumbs up, Smile ...) your project receives, the higher it appears in this list.'
 
+
+follower:
+  follows: Following
+  followers: Followers
+  follow: Follow
+  amountOfPrograms: Projects
```

I think for new items, we should stick to the new term **project**.
```suggestion
  amountOfProjects: Projects
```

Maybe it makes sense to use a global one for the term "Projects"?